api/v1beta1/hostedcluster_types.go

```diff
@@ -2100,6 +2100,7 @@ type ClusterConfiguration struct {
 	// It is used to configure the integrated OAuth server.
 	// This configuration is only honored when the top level Authentication config has type set to IntegratedOAuth.
 	// +optional
+	// +kubebuilder:validation:XValidation:rule="duration(self.tokenConfig.accessTokenInactivityTimeout).getSeconds() >= 300", message="spec.configuration.oauth.tokenConfig.accessTokenInactivityTimeout minimum acceptable token timeout value is 300 seconds"
```

Comment:
how's this been validated in standalone https://github.com/openshift/api/blob/master/config/v1/types_oauth.go#L76

Reply:
probably using an admission webhook

Reply:
can we validate that and link the source here?

Reply:
https://github.com/openshift/kubernetes/blob/c0449761f4ad0c3a585990c26aea90a1dadb82cb/openshift-kube-apiserver/admission/customresourcevalidation/oauth/validate_idp.go#L75C1-L81

Reply:
thanks!